test/src/test/java/org/corfudb/integration/LogReplicationIT.java

```diff
@@ -383,8 +392,92 @@ private void verifyNoData(Map<String, Table<StringKey, IntValue, Metadata>> tabl
         }
     }
 
+    private void verifyPreviousSnapshotStatus(LogReplicationMetadata.SyncStatus expectedStatus) throws InvocationTargetException, NoSuchMethodException, IllegalAccessException {
+
+        LogReplicationMetadata.ReplicationStatusKey key = LogReplicationMetadata.ReplicationStatusKey.newBuilder().setClusterId(REMOTE_CLUSTER_ID).build();
+        LogReplicationMetadata.SyncStatus snapshotSyncStatus = null;
+        Table<LogReplicationMetadata.ReplicationStatusKey, LogReplicationMetadata.ReplicationStatusVal, Message> replicationStatusTable;
+        replicationStatusTable = srcCorfuStore.openTable(CORFU_SYSTEM_NAMESPACE,
```

Comment:
nit: initialization and declaration can be on the same line

Reply:
Got it. Updated likewise.